app/tests/integration/nimbus/pages/summary.py

```diff
@@ -35,8 +40,28 @@ def request_review(self):
         return self.RequestReview(self)
 
     def approve(self):
-        el = self.find_element(*self._approve_request_button_locator)
+        for _ in range(45):
```

Comment:
@jrbenny35 Should this be `range(timeout_length)`?

Reply:
doh, missed one. I'll fix tomorrow